### PR TITLE
Fix docs

### PR DESCRIPTION
**Pull request by chaficnajjar**

### CODE REVIEW

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx [hunk 1]
```diff
@@ -551,7 +551,7 @@ const checkJson = await prisma.user.findMany({
   { "status": "expired", "insuranceID": 92 } // MySQL
   ```
 
-- If your array contains multiple objects, PostgreSQL will only return results if 
+- If your array contains multiple objects, PostgreSQL will only return results if
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]: This is still not a full sentence though, right?

File: content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx [hunk 2]
```diff
@@ -602,9 +602,9 @@ model Play {
   id       Int   @id @default(autoincrement())
   score    Int?  @default(0)
   playerId Int?
-  player   User? @relation(fields: playerId, references: id)
-  gameId Int?
-  game Game? @relation(fields: gameId, references: id)
+  player   User? @relation(fields: [playerId], references: [id])
+  gameId   Int?
+  game     Game? @relation(fields: [gameId], references: [id])
```
Comment (janpio) [inline content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx:607]: Do the brackets matter here? Does the formatter auto correct to this format?

File: content/200-concepts/100-components/04-introspection.mdx [hunk 3]
```diff
@@ -204,12 +204,12 @@ Prisma translates this into the following data model:
 
 ```prisma
 model User {
-  id      Int      @default(autoincrement()) @id
+  id      Int      @id @default(autoincrement())
```
Comment (janpio) [inline content/200-concepts/100-components/04-introspection.mdx:207]: This is what the formatter does to the schema?

**Review by janpio — APPROVED** [review]
<3

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx (hunk 1 above)
Comment (chaficnajjar) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]: @janpio Right, this sentence seems incomplete. Can you ask someone on your team to fix this? I don't know what the full sentence should look like.

File: content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx (hunk 2 above)
Comment (chaficnajjar) [reply content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx:607]: @janpio I didn't use the formatter but it says [here](https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#relation) that the type of `fields` and `references` should be `FieldReference[]`, so I assumed that the brackets mattered here. The brackets are also used in the rest of the docs so even if they're not required, I'm guessing that it's better to have them to make the code style across the docs more coherent.

File: content/200-concepts/100-components/04-introspection.mdx (hunk 3 above)
Comment (chaficnajjar) [reply content/200-concepts/100-components/04-introspection.mdx:207]: @janpio I didn't use the formatter but it says [here](https://www.prisma.io/docs/concepts/components/introspection#order-of-attributes) that `@id` must come before `@default`. I also used `prisma introspect` in my own project and it did place `@id` before `@default`.

Comment (janpio) [reply content/200-concepts/100-components/04-introspection.mdx:207]: Then you effectively used the same codebase as the formatter 👍

File: content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx (hunk 1 above)
Comment (mhwelander) [reply content/200-concepts/100-components/02-prisma-client/051-working-with-fields/100-working-with-json-fields.mdx]:
```suggestion
- If your filter array contains multiple objects, PostgreSQL will only return results if _all_ objects are present - not if at least one object is present.
```

File: content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx (hunk 2 above)
Comment (mhwelander) [reply content/200-concepts/100-components/02-prisma-client/055-aggregation-grouping-summarizing.mdx:607]: Fair - both work, but we have generally used square brackets in the docs. I see a few places where this isn't the case but we can clean that up.

**Comment by mhwelander** [comment]
💚 This is great!

**Pull request merged**